client/src/Home.js

```diff
@@ -1,20 +1,80 @@
-import React, { useState, useEffect } from "react";
-import Api from "./Api";
-import BudgetList from "./components/budget/BudgetList";
-function Home() {
-  const [budget, setBudget] = useState([]);
+import React, { useEffect } from "react";
+import { useLocation, useHistory } from "react-router-dom";
 
-  useEffect(() => {
-    Api.budget.index().then((res) => setBudget(res.data));
-  }, []);
+import "./Home.scss";
+import PigCoin from "./shared/pics/undraw_Savings_re_eq4w (1).png";
+import LadyMoney from "./shared/pics/undraw_investment_data_yfvt.png";
 
+function Home() {
   return (
-    <main className="container">
-      <header>
-        <h2>
-          <BudgetList budget={budget} />
-        </h2>
-      </header>
+    <main className="home">
+      <section className="home__top">
+        <h1 className="display-1">Welcome to Bulga.</h1>
```

Comment:
Don't overthink and create your own classes when not necessary. INSPECT the text in Figma. If it says its a Display Heading 1, use the h1 with the display-1 heading class from Bootstrap. Then, override these shared styles in App.scss for the design. Then it will be easy to re-use everwhere.